extensions/wikia/AdEngine3/src/setup.js

```diff
@@ -207,6 +206,7 @@ async function setupAdContext(wikiContext, consents) {
 
 		const priceFloorRule = instantConfig.get('icPrebidSizePriceFloorRule');
 		context.set('bidders.prebid.priceFloor', priceFloorRule || null);
+		context.set('bidders.ixIdentityLibrary.enabled', instantConfig.get('icIxIdentityLibrary'));
```

Comment:
I would store it as `bidders.indexExchange.identityLibrary.enabled` inside bidders shared, but whatever 🤷